client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -45,9 +45,6 @@ const availableFlows: Record< string, () => Promise< { default: Flow } > > = {
 	'readymade-template': () =>
 		import( /* webpackChunkName: "readymade-template-flow" */ './readymade-template' ),
 
-	'free-post-setup': () =>
```

Comment:
Can we remove instances of `free-post-setup` in `client/landing/stepper/declarative-flow/internals/global.scss` too, or do they refer to the step (and not the flow) ?

Also, calling the step and the flow with the same name is quite confusing 😅

Reply:
Good question!

We're doing it for both the flow name and the step name, unfortunately:

https://github.com/Automattic/wp-calypso/blob/b9df8390739329dfc45f60f38e3d6e040c886a2d/client/landing/stepper/declarative-flow/internals/components/step-route/index.tsx#L68-L74

So I prefer to keep it there, since it's very likely that some of the existing step might still need that CSS.